staging/src/k8s.io/apiserver/pkg/server/config_selfclient.go

```diff
@@ -63,6 +63,11 @@ func LoopbackHostPort(bindAddress string) (string, string, error) {
 		return "", "", fmt.Errorf("invalid server bind address: %q", bindAddress)
 	}
 
+	isIPv6 := false
+	if net.ParseIP(host).To4() == nil {
```

Comment:
This could also (like below) just be:

`isIPv6 := net.ParseIP(host).To4() == nil